api/outdated/outdated/parser.py

```diff
@@ -60,6 +60,29 @@ def _get_version(
             minor_version=semver.minor,
         )
 
+        if remote_name := next(
+            (
+                remote_name
+                for key, remote_name in settings.ENDOFLIFE_DATE_ASSOCIATIONS.items()
+                if search(key, dependency.name)
+            ),
+            None,
+        ):
+            try:
+                resp = requests.get(
+                    f"https://endoflife.date/api/{remote_name}/{release_version.release_version}.json",
+                    timeout=10,
+                )
+                data = resp.json()
+                if eol := data.get("eol"):
+                    release_version.end_of_life = parse_date(eol)
+                    release_version.save()
+            except (
+                requests.exceptions.ReadTimeout,
+                requests.exceptions.JSONDecodeError,
+            ):  # pragma: no cover
+                pass
```

Comment:
For the sake of readability, I would move this block into its own method.

Reply:
done here: https://github.com/adfinis/Outdated/pull/200/commits/e134387dd39fdee48928f8fb62e839463ca3c324